1-js/02-first-steps/11-logical-operators/article.md

```diff
@@ -258,46 +258,46 @@ if (x > 0) {
 }
 ```
 
-The variant with `&&` appears shorter. But `if` is more obvious and tends to be a little bit more readable.
+A variante com `&&` parece mais curta. Mas `if` é mais óbvio e tende a ser um pouco mais legível.
 
-So we recommend using every construct for its purpose: use `if` if we want if and use `&&` if we want AND.
+Por isso recomendamos usar cada constructo para o seu propósito: use `if` se quisermos usar if e usar `&&` se quisermos AND.
 
 ## ! (NOT)
 
-The boolean NOT operator is represented with an exclamation sign `!`.
+O operador booleano NOT é representado por um sinal de exclamação `!`.
 
-The syntax is pretty simple:
+A sintaxe é bem simples:
 
 ```js
 result = !value;
 ```
 
-The operator accepts a single argument and does the following:
+O operador aceita um único argumento e faz o seguinte:
 
-1. Converts the operand to boolean type: `true/false`.
-2. Returns the inverse value.
+1. Converte o operando em um tipo booleano: `true/false`.
+2. Retorna o valor inverso.
 
-For instance:
+Por exemplo:
 
 ```js run
-alert( !true ); // false
-alert( !0 ); // true
+alert( !true ); // falso
+alert( !0 ); // verdadeiro
 ```
 
-A double NOT `!!` is sometimes used for converting a value to boolean type:
+Um double NOT `!!` às vezes é usado para converter um valor em um tipo booleano:
 
 ```js run
-alert( !!"non-empty string" ); // true
-alert( !!null ); // false
+alert( !!"non-empty string" ); // verdadeiro
+alert( !!null ); // falso
 ```
 
-That is, the first NOT converts the value to boolean and returns the inverse, and the second NOT inverses it again. In the end, we have a plain value-to-boolean conversion.
+Ou seja, o primeiro NÃO converte o valor em booleano e retorna o inverso, e o segundo NÃO o inverte novamente. No final, temos uma conversão simples de valor em booleano.
 
-There's a little more verbose way to do the same thing -- a built-in `Boolean` function:
+Há uma maneira um pouco mais detalhada de fazer a mesma coisa -- uma função `Boolean` incorporada:
 
 ```js run
-alert( Boolean("non-empty string") ); // true
+alert( Boolean("non-empty string") ); // verdadeiro
 alert( Boolean(null) ); // false
 ```
 
-The precedence of NOT `!` is the highest of all logical operators, so it always executes first, before `&&` or `||`.
+A precedência de NOT `!` é o maior de todos os operadores lógicos, então ele sempre executa primeiro, antes de `&&` ou `||`.
```

Comment:
"...é **a** maior **entre** todos os operadores lógicos..."
"precedência" is feminine, so we use **a** not **o**.
"entre" is better than "de" in this phrase.